README.md

```diff
@@ -449,6 +455,13 @@ manager.getCredentials(new BaseCallback<Credentials, CredentialsManagerException
 ```
 
 
+5. **Clear credentials**
+Whenever you need to remove the credentials from the storage and make sure that the next call to `getCredentials` fails, you must clear them from the manager:
```

Comment:
this is too technical, maybe it's simpler to just say "when you want to log the user out".

> and make sure that the next call to `getCredentials` fails

saying it like this sounds weird. my goal is not to have the call fail, that it fails is just a side effect for being logged out